tools/rna_tools/selectsequences/selectsequences.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="SelectSequences" name="SelectSequences" version="1.0.0">
+  <description>
+    Tool to select representative sequences from a multiple sequence alignment.
```

Comment:
Galaxy will create from the name __and__ the description the text in the left-hand-side toolbox. I would remove the `useful` section from the description. Maybe name="Select sequences" description="from representatives of a multiple sequence alignment"?